modules/opensimplex/doc_classes/OpenSimplexNoise.xml

```diff
@@ -32,7 +32,7 @@
 			<argument index="1" name="height" type="int">
 			</argument>
 			<description>
-				Generate a noise image with the requested [code]width[/code] and [code]height[/code], based on the current noise parameters.
+				Generate a noise image in format [constant Image.FORMAT_L8] with the requested [code]width[/code] and [code]height[/code], based on the current noise parameters.
```

Comment:
I'd use `in [constant Image.FORMAT_L8] format` as done for the other method, to be consistent in the language.

Reply:
Thanks! I was debating this, too. I'll err on the side of consistency from now on.